cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion_test.go

```diff
@@ -106,8 +106,8 @@ func TestConvertToUseHyperKubeImage(t *testing.T) {
 			desc: "matching UnifiedControlPlaneImage sets UseHyperKubeImage to true",
 			in: &ClusterConfiguration{
 				ImageRepository:          "k8s.gcr.io",
-				KubernetesVersion:        "v1.12.2",
-				UnifiedControlPlaneImage: "k8s.gcr.io/hyperkube:v1.12.2",
+				KubernetesVersion:        "v1.13.2",
```

Comment:
can we use some of the helpers @bart0sh added to the version package?
https://github.com/kubernetes/kubernetes/commit/af07603178b6fca272106a941fdac87c957f7766#diff-533e2521f45a66cc7c4c2591b07ae7f8

you can search for the current usage that we have in kubeadm.
the idea is to use `MinimumControlPlaneVersion` + these methods.

i guess this comment applies to the rest of the changes too.